full/Dockerfile

```diff
@@ -275,6 +275,21 @@ ENV PATH=$PATH:$HOME/.cargo/bin
 
 RUN bash -lc "cargo install cargo-watch cargo-edit cargo-tree"
 
+### Docker ###
+LABEL dazzle/layer=docker
+LABEL dazzle/test=tests/tool-docker.yaml
+USER root
+# https://docs.docker.com/engine/install/ubuntu/
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce docker-ce-cli containerd.io \
+    && rm -rf /var/lib/apt/lists/* /tmp/*
+# https://github.com/wagoodman/dive
+RUN wget https://github.com/wagoodman/dive/releases/download/v0.9.2/dive_0.9.2_linux_amd64.deb \
```

Comment:
How many Gitpod users do you expect to frequently run `dive`? (I.e. is it worth the extra unpacked 4.5MB to `workspace-full` to have it pre-installed, as opposed to letting users install it themselves?)

Reply:
The value `dive` delivers is incredible considering it only adds 4.5MB to the scale. Everybody who uses `docker build` should use it to have smaller and more secure docker images with more cleanly designed layers.

Reply:
Okay, rationale makes sense and added size is relatively small. Thanks!